src/inspector/view/qml/MuseScore/Inspector/score/ScoreAccessibilityInspectorView.qml

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-Studio-CLA-applies
+ *
+ * MuseScore Studio
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import MuseScore.Inspector 1.0
+import Muse.UiComponents 1.0
+import Muse.Ui 1.0
+
+import "../common"
+
+InspectorSectionView {
+    id: root
+
+    implicitHeight: contentColumn.implicitHeight
+
+    Column {
+        id: contentColumn
+        width: parent.width
+        spacing: 8
+
+        Text {
+            text: qsTr("Score style preset")
+            color: ui.theme.fontPrimaryColor
+        }
+
+        StyledDropdown {
+            id: scoreStylePreset
+            width: parent.width
+            model: [
+                { text: qsTr("Default"), value: "Default" },
+                { text: qsTr("16mm MSN"), value: "16mm MSN" },
+                { text: qsTr("18mm MSN"), value: "18mm MSN" },
+                { text: qsTr("20mm MSN"), value: "20mm MSN" },
+                { text: qsTr("22mm MSN"), value: "22mm MSN" },
+                { text: qsTr("25mm MSN"), value: "25mm MSN" }
+            ]
```

Comment:
This model should be defined in the C++ code. Here's how:

1. [Define enum](https://github.com/musescore/MuseScore/blob/28aa0ac3c8be313c1fc0f4490cc58b505df73384/src/braille/brailletypes.h#L34-L40)
2. [Assign string values](https://github.com/musescore/MuseScore/blob/28aa0ac3c8be313c1fc0f4490cc58b505df73384/src/appshell/view/preferences/braillepreferencesmodel.cpp#L55-L74).
    - Example translator comment:<br>`//: Modified Stave Notation (MSN) with 16mm staff size. Intended for visually-impaired musicians.`
4. [Expose to QML](https://github.com/musescore/MuseScore/blob/28aa0ac3c8be313c1fc0f4490cc58b505df73384/src/appshell/view/preferences/braillepreferencesmodel.h#L50)
5. [Use in QML](https://github.com/musescore/MuseScore/blob/master/src/appshell/qml/Preferences/BraillePreferencesPage.qml#L57)
6. [Assign to dropdown](https://github.com/musescore/MuseScore/blob/cfbd6ef50e93d884a0ac0eb6e6e0e683d2c144ed/src/appshell/qml/Preferences/internal/BrailleAdvancedSection.qml#L61). (You can combine this with 4. since your QML file has direct access to the model.)